package.json

```diff
@@ -21,53 +21,60 @@
     "test": "tests"
   },
   "scripts": {
-    "build": "ember build",
-    "lint": "eslint .",
+    "build": "ember build --environment=production",
+    "lint": "npm-run-all --aggregate-output --continue-on-error --parallel lint:*",
+    "lint:js": "eslint .",
     "start": "ember serve",
-    "test": "tape node-tests/**.js",
-    "test:all": "ember try:each"
+    "test": "npm-run-all lint:* test:*",
+    "test:node": "tape node-tests/**.js",
+    "test:ember": "ember test",
+    "test:ember-compatibility": "ember try:each"
   },
   "devDependencies": {
-    "@ember/optional-features": "^0.6.3",
-    "broccoli-asset-rev": "^2.7.0",
-    "broccoli-merge-trees": "^3.0.2",
-    "ember-ajax": "^3.1.0",
-    "ember-cli": "~3.5.0-beta.1",
-    "ember-cli-dependency-checker": "^3.0.0",
-    "ember-cli-eslint": "^4.2.3",
-    "ember-cli-htmlbars": "^3.0.0",
-    "ember-cli-htmlbars-inline-precompile": "^1.0.3",
-    "ember-cli-inject-live-reload": "^1.8.2",
-    "ember-cli-qunit": "^4.3.2",
+    "@ember/optional-features": "^1.3.0",
+    "babel-eslint": "^10.1.0",
+    "broccoli-asset-rev": "^3.0.0",
+    "ember-auto-import": "^1.6.0",
+    "ember-cli": "~3.20.2",
+    "ember-cli-dependency-checker": "^3.2.0",
+    "ember-cli-inject-live-reload": "^2.0.2",
     "ember-cli-sri": "^2.1.1",
-    "ember-cli-template-lint": "^1.0.0-beta.1",
-    "ember-cli-uglify": "^2.1.0",
+    "ember-cli-uglify": "^3.0.0",
     "ember-disable-prototype-extensions": "^1.1.3",
-    "ember-export-application-global": "^2.0.0",
-    "ember-load-initializers": "^1.1.0",
+    "ember-export-application-global": "^2.0.1",
+    "ember-load-initializers": "^2.1.1",
     "ember-maybe-import-regenerator": "^0.1.6",
-    "ember-resolver": "^5.0.1",
-    "ember-source": "~3.5.0-beta.1",
-    "ember-source-channel-url": "^1.1.0",
-    "ember-try": "^1.0.0",
-    "eslint-plugin-ember": "^5.2.0",
-    "eslint-plugin-node": "^7.0.1",
+    "ember-qunit": "^4.6.0",
+    "ember-resolver": "^8.0.0",
+    "ember-source": "~3.20.2",
+    "ember-source-channel-url": "^2.0.1",
+    "ember-template-lint": "^2.9.1",
+    "ember-try": "^1.4.0",
+    "eslint": "^7.5.0",
+    "eslint-plugin-ember": "^8.9.1",
+    "eslint-plugin-node": "^11.1.0",
     "loader.js": "^4.7.0",
-    "qunit-dom": "^0.7.1",
+    "npm-run-all": "^4.1.5",
+    "qunit-dom": "^1.2.0",
     "tape": "^4.9.1"
   },
   "engines": {
-    "node": "6.* || 8.* || >= 10.*"
+    "node": "10.* || >= 12"
+  },
+  "ember": {
+    "edition": "octane"
   },
   "ember-addon": {
     "configPath": "tests/dummy/config",
     "defaultBlueprint": "ember-cli-storybook"
   },
   "dependencies": {
     "broccoli-funnel": "^2.0.2",
+    "broccoli-merge-trees": "^3.0.2",
```

Comment:
I moved this to `dependencies` as it's used in `index.js`

https://github.com/storybookjs/ember-cli-storybook/blob/ec5587cc7f09a96d58d2bf7248693e0ff3cb61df/index.js#L7